<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtTellerMachineClass</class>
 <widget class="QMainWindow" name="qtTellerMachineClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>1069</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>qtTellerMachine</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>MD</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="btn_mdlogin">
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_save">
         <property name="text">
          <string>Save To File</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>INST:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_ins">
         <property name="text">
          <string>m1705</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_subscribe">
         <property name="text">
          <string>Subscribe</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_unsubscribe">
         <property name="text">
          <string>Unsubscribe</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_tradingday">
         <property name="text">
          <string>TradingDay:N/A</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>TRADE</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>USER:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_usrname">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>PASSWORD:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_pswd">
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_tdlogin">
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string>Settlement INFO</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Confirm Seattlement INFO</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_accountQry">
         <property name="text">
          <string>Account Qry</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>INTRADAY TRADING</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>PRICE:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="sb_tdprice">
         <property name="maximum">
          <number>9999</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>THRES:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QSpinBox" name="sb_tddiff">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>5</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btn_copyPrice">
         <property name="text">
          <string>â†‘</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" rowspan="2" colspan="3">
        <widget class="QLCDNumber" name="lcd_lastPrice">
         <property name="digitCount">
          <number>4</number>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
         <property name="value" stdset="0">
          <double>8888.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="lb_ask">
         <property name="text">
          <string>N/A</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4" rowspan="2" colspan="2">
        <widget class="QPushButton" name="pushButton_4">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>CLOSE ALL</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="lb_bid">
         <property name="text">
          <string>N/A</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>SHORT</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="pushButton_3">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>255</green>
               <blue>127</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Light">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>127</red>
               <green>255</green>
               <blue>191</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Midlight">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>63</red>
               <green>255</green>
               <blue>159</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Dark">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>127</green>
               <blue>63</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Mid">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>170</green>
               <blue>84</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Text">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="BrightText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>255</green>
               <blue>127</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Shadow">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="AlternateBase">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>127</red>
               <green>255</green>
               <blue>191</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ToolTipBase">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>220</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ToolTipText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>255</green>
               <blue>127</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Light">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>127</red>
               <green>255</green>
               <blue>191</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Midlight">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>63</red>
               <green>255</green>
               <blue>159</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Dark">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>127</green>
               <blue>63</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Mid">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>170</green>
               <blue>84</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Text">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="BrightText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>255</green>
               <blue>127</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Shadow">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="AlternateBase">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>127</red>
               <green>255</green>
               <blue>191</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ToolTipBase">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>220</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ToolTipText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>127</green>
               <blue>63</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>255</green>
               <blue>127</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Light">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>127</red>
               <green>255</green>
               <blue>191</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Midlight">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>63</red>
               <green>255</green>
               <blue>159</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Dark">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>127</green>
               <blue>63</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Mid">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>170</green>
               <blue>84</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Text">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>127</green>
               <blue>63</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="BrightText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>127</green>
               <blue>63</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>255</green>
               <blue>127</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>255</green>
               <blue>127</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Shadow">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="AlternateBase">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>255</green>
               <blue>127</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ToolTipBase">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>220</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ToolTipText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="text">
          <string>LONG</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qttellermachine.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_mdlogin</sender>
   <signal>clicked()</signal>
   <receiver>qtTellerMachineClass</receiver>
   <slot>mdLogin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_subscribe</sender>
   <signal>clicked()</signal>
   <receiver>qtTellerMachineClass</receiver>
   <slot>mdSubscribe()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_tdlogin</sender>
   <signal>clicked()</signal>
   <receiver>qtTellerMachineClass</receiver>
   <slot>traderLogin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_accountQry</sender>
   <signal>clicked()</signal>
   <receiver>qtTellerMachineClass</receiver>
   <slot>traderAccountQry()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_unsubscribe</sender>
   <signal>clicked()</signal>
   <receiver>qtTellerMachineClass</receiver>
   <slot>mdUnsubscribe()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_copyPrice</sender>
   <signal>clicked()</signal>
   <receiver>qtTellerMachineClass</receiver>
   <slot>copyPrice()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>qtTellerMachineClass</receiver>
   <slot>tradeLong()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>qtTellerMachineClass</receiver>
   <slot>tradeShort()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>qtTellerMachineClass</receiver>
   <slot>confirmSettlementInfo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>qtTellerMachineClass</receiver>
   <slot>qrySettlementInfo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>mdLogin()</slot>
  <slot>mdSubscribe()</slot>
  <slot>traderLogin()</slot>
  <slot>traderAccountQry()</slot>
  <slot>mdUnsubscribe()</slot>
  <slot>copyPrice()</slot>
  <slot>tradeLong()</slot>
  <slot>tradeShort()</slot>
  <slot>confirmSettlementInfo()</slot>
  <slot>qrySettlementInfo()</slot>
 </slots>
</ui>
